content/en/docs/reference/specs.md

```diff
@@ -34,7 +34,7 @@ CTAP is implemented by authenticator and device vendors and abstracts away all o
 
 ***Current Version***: [CTAP 2.1](https://fidoalliance.org/specs/fido-v2.1-ps-20210615/fido-client-to-authenticator-protocol-v2.1-ps-errata-20220621.html)
 
-***Next Version***: CTAP 2.2
+***Next Version***: CTAP 2.2 [(Review Draft 01)](https://fidoalliance.org/specs/fido-v2.2-rd-20230321/fido-client-to-authenticator-protocol-v2.2-rd-20230321.html)
```

Comment:
Wait, actually why not make the link "CTAP 2.2 (Review Draft 01)" so it's all clickable?

Reply:
FINE MATT